source/beamline/REIXS/REIXSXESMCPDetector.cpp

```diff
@@ -343,3 +371,13 @@ void REIXSXESMCPDetector::acquisitionCancelledHelper(){
 
 	emit imageDataChanged();
 }
+
+void REIXSXESMCPDetector::toggleVeto(bool on)
+{
+	if (vetoControl_->isConnected() && vetoStateControl_->isConnected()) {
+		if (on)
+			vetoControl_->move(0); // veto off is to enable updating
+		else
+			vetoControl_->move(1); // veto on is to disable updating
+	}
+}
```

Comment:
I think for clarity of this particular method we should not use "toggle".  Since we are specifically stopping the detector from counting photons, I would suggest a name such as `disableMCPCountUpdate(bool)`.  It also makes the boolean value map to the control value a little better as well because, if I'm reading correctly, the function would look like this,

```c++
if (disableUpdate)
    vetoControl_->move(1);
else
    vetoControl_->move(0);
```